### DISCUSSION TITLE
Is the timestamp option in ffmpeg available in encoders

### DISCUSSION THREAD

**Discussion by Russsgithub** [savonet/liquidsoap#3930] [General]
I'm trying to add a record timestamp ffmpeg option in an encoder but get unrecongized option.
Does anyone know if it is possible ?

```
encoder = %ffmpeg(
  format="mp4",
  %audio(codec="aac", b="320k", samplerate=48000, q=4),
  %video(codec="libx264",width=1920, height=1080,b="5000k", preset="veryfast"),
  movflags="frag_keyframe+empty_moov",
  timestamp=now
)
```

Thanks

.......

Update.

-timestamp in ffmpeg has been depreciated.

The replacment seems to be to add "creation_time" as metadata.

**Comment by vitoyucepi** [comment]
Hi @Russsgithub,

> -timestamp in ffmpeg has been depreciated.

I could not find any information about timestamp deprecation.

> -timestamp date (output)
> Set the recording timestamp in the container.
> 
> date must be a date specification, see [(ffmpeg-utils)the Date section in the ffmpeg-utils(1) manual](https://ffmpeg.org/ffmpeg-utils.html#date-syntax).

May I ask you to create a feature request?

**Comment by Russsgithub** [comment]
If I run an ffmpeg command with -timestamp now in it I get time stamp is depreciated use -metadata "creation_time" .If I search I find nothing either.Will do, thanks @VitoOn 20 May 2024 21:20, Vito ***@***.***> wrote:
Hi @Russsgithub,

-timestamp in ffmpeg has been depreciated.

I could not find any information about timestamp deprecation.

-timestamp date (output)
Set the recording timestamp in the container.
date must be a date specification, see (ffmpeg-utils)the Date section in the ffmpeg-utils(1) manual.

May I ask you to create a feature request?
